templates/zerver/register.html

```diff
@@ -227,7 +227,7 @@ <h1>You&rsquo;re almost there.</h1>
                 </div>
                 {% endif %}
                 <div class="register-button-box">
-                    <button class="register-button" type="submit">{{ _('Sign up') }}</button>
+                    <button class="register-button" id="register-button-id" type="submit" disabled>{{ _('Sign up') }}</button>
```

Comment:
I would name the button id `user-register-button` or something similar. It does not make much sense to have `id` in the id name since it's obvious it is an id :)

Reply:
I couldn't find a better id name at that time(I am very bad at naming xD). I had changed it.